expression/integration_test.go

```diff
@@ -696,6 +696,15 @@ func (s *testIntegrationSuite2) TestMathBuiltin(c *C) {
 	result.Check(testkit.Rows("100000000000000 1000000000000000 100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"))
 	result = tk.MustQuery("SELECT ROUND(1e-14, 1), ROUND(1e-15, 1), ROUND(1e-308, 1)")
 	result.Check(testkit.Rows("0 0 0"))
+	result = tk.MustQuery("SELECT round(49.99999, -2), round(50, -2), round(50.00001, -2)")
+	result.Check(testkit.Rows("0 100 100"))
+	result = tk.MustQuery("SELECT round(123456789, -5), round(2146213728964879326, -15) ")
+	result.Check(testkit.Rows("123500000 2146000000000000000"))
+	// MySQL 8 will report an error if round result overflows
+	rs, err = tk.Exec("select round(18446744073709551615, -19);")
+	c.Assert(err, NotNil)
+	terr = errors.Cause(err).(*terror.Error)
+	c.Assert(terr.Code(), Equals, errors.ErrCode(mysql.ErrDataOutOfRange))
```

Comment:
```
[2021-08-19T15:56:19.448Z] FAIL: integration_test.go:413: testIntegrationSuite2.TestMathBuiltin
[2021-08-19T15:56:19.448Z] 
[2021-08-19T15:56:19.448Z] integration_test.go:705:
[2021-08-19T15:56:19.448Z]     c.Assert(err, NotNil)
[2021-08-19T15:56:19.448Z] ... value = nil
```

tests failed by above. I think you can take a look at "// for pow" part to see how to validate errors.

Reply:
```go
	rs, err = tk.Exec("SELECT POW(0, -1);")
	c.Assert(err, IsNil)
	_, err = session.GetRows4Test(ctx, tk.Se, rs)
	c.Assert(err, NotNil)
	terr = errors.Cause(err).(*terror.Error)
	c.Assert(terr.Code(), Equals, errors.ErrCode(mysql.ErrDataOutOfRange))
	c.Assert(rs.Close(), IsNil)
```

Reply:
> ```
> [2021-08-19T15:56:19.448Z] FAIL: integration_test.go:413: testIntegrationSuite2.TestMathBuiltin
> [2021-08-19T15:56:19.448Z] 
> [2021-08-19T15:56:19.448Z] integration_test.go:705:
> [2021-08-19T15:56:19.448Z]     c.Assert(err, NotNil)
> [2021-08-19T15:56:19.448Z] ... value = nil
> ```
> 
> tests failed by above. I think you can take a look at "// for pow" part to see how to validate errors.

thanks.
i learned validate error like above from the other tidb test files. i will investigate to learn more.

sorry and can't image the log clearly show why it fails but i did not notice..
the first is i need more careful, the second is the log is kind of noisy